### PR TITLE
Implement derived series for V2 API.

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

File: proto/v2/observation.proto [hunk 1]
```diff
@@ -57,9 +57,24 @@ message FacetFilter {
...
+  // 2. op: ObservationPeriod
+  // 3. ut: Unit
+  // 4. sf: ScalingFactor
+  //
+  // For example:
+  // 1. Person_Count - Person_Count_Female
+  // 2. Person_Count_Female / Person_Count[mm=CensusACS5yrSurvey;op=P5Y]
+  // 3. Person_Count - Person_Count_Female - Person_Count_Male[op=P5Y]
+  // 4. (Person_Count_Male - Person_Count_Female) / Person_Count
   DcidOrExpression variable = 1;
```
Comment (shifucun) [inline proto/v2/observation.proto:76]: Oh, can we have 3 fields in DcidOrExpression: dcids, expression, formula? 

Expression is now defined as "graph expression" (like in entity), so we should have a separate one for algebra expression (formula)? Also for variable, we can use graph expression to get a list of variables (like the child sv of an svg).

Comment (Spaceenter) [reply proto/v2/observation.proto:76]: SG - done

File: proto/v2/observation.proto [hunk 2]
```diff
@@ -57,9 +58,24 @@ message FacetFilter {
 
 // Generic observation request
 message ObservationRequest {
-  // A list of entity DCIDs
+  // A list of statistical variable DCIDs, or a formula expression.
+  //
+  // Supported operators for the formula expression: +, -, *, /, along with parentheses.
```
Comment (shifucun) [inline proto/v2/observation.proto:63]: nit: maybe move the examples to line 28 where formula is defined?

**Review by shifucun — APPROVED** [review]
thanks for the update!

File: proto/v2/observation.proto (hunk 2 above)
Comment (Spaceenter) [reply proto/v2/observation.proto]: done

**Pull request merged**